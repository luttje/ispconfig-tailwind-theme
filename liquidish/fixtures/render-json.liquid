{% render_json 'render-json-component.liquid', {
    "slot": "{{ logout_txt }} {{ cpuser }}",
    "attributes": [
        ["id", "logout-button"],
        ["data-load-content", "login/logout.php"]
    ]
} %}
