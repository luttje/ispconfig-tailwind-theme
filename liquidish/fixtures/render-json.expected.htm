<button class="rounded bg-red-500 text-white py-2 px-4 hover:bg-red-600 uppercase shadow border-bottom-[#980b13] hover:border-bottom-[#800a10] whitespace-nowrap text-sm font-medium self-start "
        id="logout-button"
        data-load-content="login/logout.php"
        >
        {tmpl_var name="logout_txt"} {tmpl_var name="cpuser"}
</button>
